src/main/java/com/minecolonies/sounds/ModSoundEvents.java

```diff
@@ -0,0 +1,41 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.lib.Constants;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+/**
+ * Registering of sound events for our colony.
+ */
+public class ModSoundEvents
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1118)